README.rst

```diff
@@ -0,0 +1,93 @@
+.. figure:: http://genomicsandhealth.org/files/logo_ga.png
+	    :alt: Image
+
+
+Schemas for the Data Working Group
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+|Build Status| |Docs|
+
+The `Global Alliance for Genomics and Health
+<http://genomicsandhealth.org/>`__ is an international coalition,
+formed to enable the sharing of genomic and clinical data.
+
+The `Data Working
+Group <http://genomicsandhealth.org/our-work/working-groups/data-working-group>`__
+concentrates on data representation, storage, and analysis, including
+working with platform development partners and industry leaders to
+develop standards that will facilitate interoperability.
+
+Each area of genomics and health has a dedicated team working to define
+those standards.
+
+Reads Task Team
+@@@@@@@@@@@@@@@
+
+The `Reads Task
+Team <https://groups.google.com/forum/#!forum/dwgreadtaskteam>`__ is
+focused on standards for accessing genomic read data -- collections of
+primary data collected from sequencing machines.
+
+The team will deliver:
+
+1. Data model. An abstract, mathematically complete and precise model of
```

Comment:
These numbers are no longer on their own line in RST:  https://github.com/reece/schemas/blob/documentation/README.rst